apps/app-article/src/components/article-content.js

```diff
@@ -70,14 +71,14 @@ class Content extends Component {
         );
     }
 
-    renderFactBox(block, key) {
+    renderGenericBox(block, key) {
         return (
-            <div className={"factBox"} key={key} id={"factBox-" + key}>
-                <div className={"fakta"}>FAKTA</div>
-                <h4>{block.factBox.title}</h4>
+            <div className={`genericBox genericBox-border-${getBrandValueParam()}`} key={key} id={"genericBox-" + key}>
+                <div className={"fakta"}>{block.box && block.box.headline}</div>
```

Comment:
Two things:

Let's rename the class `"fakta"` to `"genericBox-headline"` or something.

If `box.type === "fact"`, we should show a default headline `"FAKTA"`.